supplier_calendar/models/__init__.py

```diff
@@ -0,0 +1,6 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```

Comment:
agpl stil here, but you can just remove it in this case, copyrights are usually not neede in init files.